requirements/requirements.txt

```diff
@@ -39,7 +39,7 @@ pytz==2014.10
 PyYAML==3.10
 raven==3.1.17
 requests==2.0.0
-south==0.7.3
+south==0.8.0
```

Comment:
oh nice, so the higher version just allows south to pick up the change then. will update